TKGame/Debug.cs

```diff
@@ -88,7 +91,7 @@ public static void Initialize()
         /// <summary>
         /// Load/Stylize all debug UI elements
         /// </summary>
-        public static void LoadContent()
+        public static void LoadContent(VerticalStackPanel VSP)
```

Comment:
Why was this call changed? I don't think we want to be passing every stack panel in like this